app/src/main/java/com/example/proptit_protify/data/Track.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.proptit_protify.data
+
+data class Track(
+    val album: AlbumX,
+    val artist: ArtistX,
+    val available_countries: List<String>,
+    val bpm: Double,
+    val contributors: List<Contributor>,
+    val disk_number: Int,
+    val duration: Int,
+    val explicit_content_cover: Int,
+    val explicit_content_lyrics: Int,
+    val explicit_lyrics: Boolean,
+    val gain: Double,
+    val id: Int,
```

Comment:
gson is capable of smart bindings from your defined pojo class to your actuall Json object. so you are safe to remove unused stuffs

Reply:
Em cũng chưa rõ là mình cần dùng những gì sau này nên tạm thời em sẽ để đó đến khi kết thúc project được không ạ